pulser/simulation/simulation.py

```diff
@@ -283,6 +332,9 @@ def build_coeffs_ops(basis: str, addr: str) -> list[list]:
         # Time independent term:
         if self.basis_name == 'digital':
             qobj_list = []
+        elif self.basis_name == 'XY':
+            # XY Interaction Terms
+            qobj_list = [make_interaction_xy_term()] if self._size > 1 else []
```

Comment:
(See comment above) this can be a single instruction if there's a single method for the time-independent interaction term